projects/skia/build.sh

```diff
@@ -35,71 +63,33 @@ $SRC/depot_tools/gn gen out/Fuzz_mem_constraints\
     is_debug=false
     extra_cflags_c=["'"$CFLAGS_ARR"'"]
     extra_cflags_cc=["'"$CXXFLAGS_ARR"'","-DIS_FUZZING","-DIS_FUZZING_WITH_LIBFUZZER"]
+    extra_ldflags=["'"$LDFLAGS_ARR"'"]
+    skia_use_egl=true
     skia_use_system_freetype2=false
     skia_use_fontconfig=false
     skia_enable_gpu=true
-    skia_enable_skottie=false
-    extra_ldflags=["-lFuzzingEngine", "'"$CXXFLAGS_ARR"'"]'
+    skia_enable_skottie=false'
 
 $SRC/depot_tools/gn gen out/Fuzz\
     --args='cc="'$CC'"
     cxx="'$CXX'"
     is_debug=false
     extra_cflags_c=["'"$CFLAGS_ARR"'"]
     extra_cflags_cc=["'"$CXXFLAGS_ARR"'","-DIS_FUZZING_WITH_LIBFUZZER"]
+    extra_ldflags=["'"$LDFLAGS_ARR"'"]
+    skia_use_egl=true
     skia_use_system_freetype2=false
     skia_use_fontconfig=false
     skia_enable_gpu=true
-    skia_enable_skottie=true
-    extra_ldflags=["-lFuzzingEngine", "'"$CXXFLAGS_ARR"'"]'
-
-
-$SRC/depot_tools/ninja -C out/Fuzz_mem_constraints image_filter_deserialize
-
-# Don't build these fuzzers for AFL since it crashes on startup.
-# This would cause a build breakage now that AFL has build checks.
-# See https://github.com/google/oss-fuzz/issues/1338 for more details.
-if [ "$FUZZING_ENGINE" == "libfuzzer" ]
-then
-  $SRC/depot_tools/ninja -C out/Fuzz_mem_constraints api_raster_n32_canvas api_mock_gpu_canvas
-fi
-
-set +e
-for f in out/Fuzz_mem_constraints/*; do
-  # Remove unnecessary dependencies that aren't on runner containers.
-  # Libraries found through trial and error (ldd command also helpful).
-  # Note: GPU is also with the mem constraints options.
-  patchelf --remove-needed libGLU.so.1 $f
-  patchelf --remove-needed libGL.so.1 $f
-  patchelf --remove-needed libX11.so.6 $f
-done
-set -e
+    skia_enable_skottie=true'
 
+$SRC/depot_tools/ninja -C out/Fuzz_mem_constraints image_filter_deserialize api_raster_n32_canvas api_mock_gpu_canvas
```

Comment:
@kjlubick I think we can get rid of this and just do one build right?

Reply:
Right